src/components/Tickets/CreateTicket.js

```diff
@@ -0,0 +1,266 @@
+                        {item.name}
+                      </div>
+                    }
+                    inputProps={{ className: "autocomplete-input" }}
+                    value={this.state.projectValue}
+                    onChange={e => this.setState({ projectValue: e.target.value })}
+                    onSelect={value => {
+                      this.setState({
+                        projectValue: value,
+                        selectedProject: this.state.projects.filter(project => project.name === value),
+                      }, this.getPhases(this.state.projects.filter(project => project.name === value)))
+                    }}
+                  />
+                </div>
+                <div className="autocomplete-field">
+                  {this.state.projectValue && (
+                    <React.Fragment>
+                      <label htmlFor="phases">Phase</label><br></br>
+                      <Autocomplete
+                        id="phases"
+                        items={this.state.phases}
+                        getItemValue={item => item.path}
+                        shouldItemRender={(item, value) => item.path.toLowerCase().indexOf(value.toLowerCase()) > -1}
+                        renderItem={(item, highlighted) =>
+                          <div key={`${item.phaseId}-${item.ticketId}`}>
+                            {item.path}
+                          </div>
+                        }
+                        value={this.state.phaseValue}
+                        inputProps={{ className: "autocomplete-input" }}
+                        onChange={e => this.setState({ phaseValue: e.target.value })}
+                        onSelect={value => {
+                          this.setState({
+                            phaseValue: value,
+                            selectedPhase: this.state.phases.filter(phase => phase.path === value),
+                          })
+                        }}
+                      />
+                    </React.Fragment>
+                  )}
+                </div>
+                  {this.state.phaseValue && (
+                    <div>
+                      <label htmlFor="summary">Summary</label>
+                      <input
+                        className="form-control"
+                        type="text"
+                        id="summary"
+                        onChange={(e) => this.setState({ summary: e.target.value })}
+                        required
+                      ></input>
+                    </div>
+                  )}
+                  {this.state.summary && (
+                    <div>
+                      <label htmlFor="budget-hours">Budget Hours</label>
+                      <input
+                        type="number"
+                        id="budget-hours"
+                        className="form-control"
+                        onChange={(e) => this.setState({ budget: e.target.value })}
+                        required
```

Comment:
Might be nice to add some basic constraints to this, since time can be expressed in a lot of ways! This will disallow negative numbers, add a basic stepper, and add a placeholder that shows that this isn't an integer:

![image](https://user-images.githubusercontent.com/714017/103658739-9ebf4f00-4f39-11eb-8287-a598d4f0e455.png)


```suggestion
                        required
                        min="0"
                        step="0.1"
                        placeholder="0.1"
```